cmd/ttn-lw-cli/commands/users.go

```diff
@@ -40,22 +40,26 @@ func userIDFlags() *pflag.FlagSet {
 	return flagSet
 }
 
-var errNoUserID = errors.DefineInvalidArgument("no_user_id", "no user ID set")
+var errNoUserID = errors.DefineInvalidArgument("no_user_id", "no user ID set, please specify one")
 
-func getUserID(flagSet *pflag.FlagSet, args []string) *ttnpb.UserIdentifiers {
+func getUserID(flagSet *pflag.FlagSet, args []string) (*ttnpb.UserIdentifiers, error) {
 	var userID string
+	var err error
 	if len(args) > 0 {
 		if len(args) > 1 {
 			logger.Warn("multiple IDs found in arguments, considering only the first")
 		}
 		userID = args[0]
 	} else {
-		userID, _ = flagSet.GetString("user-id")
+		userID, err = flagSet.GetString("user-id")
```

Comment:
No need to do this. It only returns `err` if the flag doesn't exist (which it always does) or if it can't be converted to string (which it already is).